index.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/styles.css">
+    <link rel="stylesheet" href="index.css">
+    <link rel="stylesheet" href="vendor/bootstrap-5.1.3-dist/css/bootstrap.css">
+    <title>Ernest-Kell's Tribute Page</title>
+</head>
+<body>
+    <main id="main">
+        <h1 id="title">Ernest Kell Tribute page</h1>
+        <div id="img"> 
+            <img id="image" src="ernekell.png" alt="Kell, up right, and Ernest, down. This is a screenshot taken during microverse collaborative coding challenges">
+            <figcaption id="img-caption"> Kell, up right, and Ernest, down. This is a screenshot taken during microverse collaborative coding challenges</figcaption>
+        </div>
+        <section id="tribute-info">
+            <h3 id="tribute-info">Here's a time line of Dr. Ernest Kell's life:</h3>
+
+            <ul>
+            <li><strong>1990</strong>Born in Blantyre, Malawi </li>
+
+            <li>
+              <strong>2008</strong> - Leaves his family to attend the
+              University of Malawi, where he graduated with a bachelor's degree in Computer science four years after.
+              
+            </li>
+            <li>
+              <strong>2013</strong> - He Joined a small company that creates software programs related 
+              to health where he worked as an intern in the IT department for one year and three months.  
+              
+            </li>
```

Comment:
Please do not jump or leave line spaces anyhow unless when you want to indicate different sections of your program. It is a very good practice to only leave a line space or spaces when demarcating sections of your program.